pkg/io/video/broadcast.go

```diff
@@ -0,0 +1,75 @@
+package video
+
+import (
+	"fmt"
+	"image"
+
+	"github.com/pion/mediadevices/pkg/io"
+)
+
+var errEmptySource = fmt.Errorf("Source can't be nil")
+
+// Broadcaster is a specialized video broadcaster.
+type Broadcaster struct {
+	ioBroadcaster *io.Broadcaster
+}
+
+type BroadcasterConfig struct {
+	Core *io.BroadcasterConfig
+}
+
+// NewNewBroadcaster creates a new broadcaster.
```

Comment:
I think we should have a comment that the source reader should drop frames when there are more than single frame between two `Read` calls. (https://github.com/pion/mediadevices/pull/214/files#r489883203)

Reply:
Yep, makes sense.

Reply:
Updated.